### DISCUSSION TITLE
[Prettier Plugin] Sort classes in nunjucks

### DISCUSSION THREAD

**Discussion by njfamirm** [tailwindlabs/tailwindcss#11949] [Ideas]
Hello, Greeting!

When I code in the html, I use prettier and the ability to sort classes is wonderful and cleans the code.
But I don't have this feature in [nunjucks](https://mozilla.github.io/nunjucks/)! I use it in [11ty](https://www.11ty.dev/).

It would be great if there was such a plugin.
I would be happy if I could help and start building it

**Comment by coxde** [comment]
I'm also using Nunjucks and Tailwind to build my website, it would be nice to see this feature!

**Comment by njfamirm** [comment]
this feature support using prettier tailwind plugin
